tests/src/Kernel/VocabularyReferenceFieldsManagerTest.php

```diff
@@ -427,15 +429,24 @@ public function testEntityFormDisplayAlter(): void {
       'association_two_94ab077978',
     ], array_keys($render_form_display->getComponents()));
 
-    $this->assertArraySubset([
+    $this->assertSame([
```

Comment:
Use `assertEquals` in all of these. The scope is to check the content of the array, not the exact order.